app/src/observability/observability/__init__.py

```
@@ -0,0 +1,158 @@
+
+from .config import ObservabilityConfig
+from .sink import Sink
+
+__all__ = ["get_logger", "configure", "log_context", "ObservabilityConfig", "Sink"]
+
+_TAG = "_observability_managed"
+HANDLER_TAG = "sink"
+SAFETY_TAG = "safety"
+
+_context_var: contextvars.ContextVar[dict] = contextvars.ContextVar(
+    "observability_context", default={}
```

Ruff (0.15.12)

[warning] Do not use mutable data structures for `ContextVar` defaults

(B039)